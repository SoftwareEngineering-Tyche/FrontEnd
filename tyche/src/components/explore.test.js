import { render, screen } from "@testing-library/react"
import Explore from "./explore";
import { shallow, mount } from 'enzyme';

test('intial explore', () => {
    const wrapper = shallow(<Explore />);
    expect(wrapper).toMatchSnapshot();
});