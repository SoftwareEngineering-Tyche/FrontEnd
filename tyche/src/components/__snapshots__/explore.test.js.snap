// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`intial explore 1`] = `ShallowWrapper {}`;
